src/big/big_decimal.cr

```diff
@@ -312,28 +312,62 @@ struct BigDecimal < Number
   def to_s(io : IO) : Nil
     factor_powers_of_ten
 
-    s = @value.to_s
-    if @scale == 0
-      io << s
-      return
+    cstr = LibGMP.get_str(nil, 10, @value)
+    length = LibC.strlen(cstr)
+    buffer = Slice.new(cstr, length)
+
+    # add negative sign
+    if buffer[0]? == 45 # '-'
```

Comment:
```suggestion
    if buffer[0]? === '-'
```
I'm pretty sure this doen't add any significant overhead and saves you from annotating the character in a comment ;)

Reply:
“===” different from ruby syntax ？